docs/pages/clients/installation.md

```diff
@@ -13,7 +13,7 @@ import ConfigurationDiff from '~/components/plugins/ConfigurationDiff';
 
 The easiest way to get started is to initialize a new project by executing the following command:
 
-<InstallSection packageName="expo-development-client" cmd={["npx crna -t with-development-client"]} hideBareInstructions />
+<InstallSection packageName="expo-development-client" cmd={["npx crna -t https://github.com/lukmccall/with-dev-launcher"]} hideBareInstructions />
```

Comment:
I've created a PR to expo/examples. So I believe we should merge https://github.com/expo/examples/pull/236 and not pointing to my repository 😜